main.py

```diff
@@ -50,6 +51,5 @@ def fahren_to_cel(temprutere_in_fahren):  # Take temperature in Fahrenheit and c
     elif answer in ['fahrenheit', 'fahren', 'far', '-f']:  # Convert from Fahrenheit to Celsius
         fahrenheit = float(input("Provide tempruture in Fahrenheit: "))
         print(fahren_to_cel(fahrenheit))
-    
     else:
-        pass
+        print("You provide nothing :) ")
```

Comment:
@MariaMaximova, :D I think here is misunderstanding: here user did not provide nothing, but provided something not equeal to fahrenheit nor celsius, so I think it is better to say "Cannot recognize your input, please try again" and make the whole if-else in the loop, so if the input is incorrect, the user will be asked again to enter a value.